Hymns/Audio Player/AudioPlayerView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import AVFoundation
+import Combine
+
+enum PlaybackState: Int {
+    case waitingForSelection
+    case buffering
+    case playing
+}
+
+struct AudioView: View {
+
+    let player = AVPlayer()
+    let item: URL?
+    fileprivate let seekDuration: Float64 = 5
+    fileprivate var playerCurrentTime: Float64 {
+        CMTimeGetSeconds(self.player.currentTime())
+    }
+
+    @State var currentlyPlaying = true
+    @State var playbackState: PlaybackState = .waitingForSelection
+
+    var body: some View {
+        VStack {
+            AudioPlayerControlsView(player: player,
+                                    timeObserver: PlayerTimeObserver(player: player),
+                                    itemObserver: PlayerItemObserver(player: player), playbackState: $playbackState)
+            HStack(spacing: 30) {
+                //Reset button currently needed for when you tab to chords, piano, or guitar....
+                Button(action: {
+                    if self.currentlyPlaying {
+                        self.currentlyPlaying.toggle()
+                    }
```

Comment:
self.currentlyPlaying = false seems like what you are doing here

Reply:
Addressed in commit 3678e3a